scorecard.js

```diff
@@ -0,0 +1,69 @@
+
+class ScoreCard {
+  //can only have properties in a class, not class consts as inside an object.
+  GUTTER_GAME = [
+    [0, 0],
+    [0, 0],
+    [0, 0],
+    [0, 0],
+    [0, 0],
+    [0, 0],
+    [0, 0],
+    [0, 0],
+    [0, 0],
+    [0, 0],
+  ];
+  
+  PERFECT_GAME = [
+    [10],
+    [10],
+    [10],
+    [10],
+    [10],
+    [10],
+    [10],
+    [10],
+    [10],
+    [10, 10, 10],
+  ];
+  
+  arraysAreIdentical(arr1, arr2){
+    return JSON.stringify(arr1) === JSON.stringify(arr2);
+}
+s
+//JSON.stringify will convert the object into a string
+
+  constructor(frames) {
```

Comment:
This is really clever -- to see weed out the perfect and gutter games by checking if they are the same arrays throughout